reactDashboard/src/components/HeatMap/Heatmap.jsx

```diff
@@ -318,11 +315,9 @@ function Heatmap({
           <tr>
             <th>
               <div>{areaTitle()}</div>
-              <div className="subheading">{areaSubTitle()}</div>
             </th>
             <th>
               <div>{valueTitle()}</div>
```

Comment:
Now that these cells only have a single line, there's no need for the div

Reply:
This is still pending

Reply:
refactored